### DISCUSSION TITLE
Problem with conda environment then using pythontex

### DISCUSSION THREAD

**Discussion by spiralhead** [James-Yu/LaTeX-Workshop#3106] [Q&A]
Cannot activate conda environment when using latex workshop. Just can't workout how to set up environment activation before using pythontex.

**Comment by tamuratak** [comment]
This issue is not related to the extension.

We don't know nothing about conda.

**Comment by tamuratak** [comment]
Please fill out the [issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md) next time.
